File: Servidor/AtHomeTFGCarlosHernandez/src/main/java/org/tfg/athometfgcarloshernandez/spring/model/response/PantallaEstadosResponseDTO.java
package org.tfg.athometfgcarloshernandez.spring.model.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.tfg.athometfgcarloshernandez.spring.model.UsuarioCasaDTO;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class PantallaEstadosResponseDTO {
    private String estado;
    private String colorEstado;
    private String colorUsuario;
    private Integer idCasa;
    private String nombreCasa;
    private String direccion;
    private String codigoInvitacion;
    private List<UsuarioCasaDTO> usuariosCasa;
    private List<String> estadosDisponibles;
    private String accessToken;
    private String refreshToken;
}

File: Cliente/InHomeTFGAndroidCarlosHernandez/app/src/main/java/com/example/inhometfgandroidcarloshernandez/data/model/response/PantallaEstadosResponseDTO.kt
package com.example.inhometfgandroidcarloshernandez.data.model.response

import com.example.inhometfgandroidcarloshernandez.data.model.UsuarioCasaDTO


data class PantallaEstadosResponseDTO(
    val estado: String = "",
    val colorEstado: String = "",
    val colorUsuario: String = "",
    val idCasa: Int = 0,
    val nombreCasa: String = "",
    val direccion: String = "",
    val codigoInvitacion: String = "",
    val usuariosCasa: List<UsuarioCasaDTO> = emptyList(),
    val estadosDisponibles: List<String> = emptyList(),
    val accessToken: String = "",
    val refreshToken: String = ""
)